routers/web/repo/issue.go

```diff
@@ -356,6 +356,17 @@ func issues(ctx *context.Context, milestoneID, projectID int64, isPullOption uti
 		labels = append(labels, orgLabels...)
 	}
 
+	if !ctx.FormBool("archived") {
```

Comment:
Hmm… That is now a far more complicated approach than I imagined.
I think this is one of the features that is better implemented in the frontend:
Simply give **all** labels to the frontend as before, add a `data-is-archived-label="true"` if the label is archived, and then run JS over it that is executed once on page load and then whenever the checkbox changes its state that toggles the `gt-hidden` class for all elements in `document.querySelectorAll('[data-is-archived=true]')` (or whatever the exact syntax is) depending on if the checkbox is false or not.
Then there is no need to torment the backend with requests or to do some URL trickery anymore.

Reply:
we can discuss on this approach.

Reply:
Done

Reply:
As discussed I have moved this approach into frontend side.

* Updates
Label issue action updated
<img width="409" alt="image" src="https://github.com/go-gitea/gitea/assets/80308335/f0c6eeff-83e7-4845-844f-d304421dc543">